src/components/Map/CircleComponent.tsx

```diff
@@ -0,0 +1,31 @@
+import L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Circle, Marker } from 'react-leaflet';
+
+import './CircleComponent.scss';
+
+function CircleComponent({ center, radius, text, color }) {
+  const text1 = L.divIcon({ className: 'icon', html: text });
+
+  return (
+    <Circle
+      center={center}
+      radius={radius}
+      fillColor={color}
+      color={color}
+      className='circle'>
```

Comment:
```suggestion
>
```
This doesn't make any style changes